File: MySQL-NET-REST-API/DocumentManagement.Data/Resources/WorkflowLogResource.cs
namespace DocumentManagement.Data.Resources
{
    public class WorkflowLogResource : ResourceParameter
    {
        public WorkflowLogResource() : base("WorkflowInstanceStatus")
        {
        }
        public string WorkflowId { get; set; }
        public WorkflowInstanceStatus WorkflowInstanceStatus { get; set; }
        public string DocumentId { get; set; }
    }
}

File: AngularFrontend/src/app/core/domain-classes/workflow-log-resource.ts
import { ResourceParameter } from './resource-parameter';
import { WorkflowInstanceStatus } from './workflow-instance-status.enum';

export class WorkflowLogResource extends ResourceParameter {
  workflowId: string = '';
  documentId: string = '';
  workflowInstanceStatus: WorkflowInstanceStatus = WorkflowInstanceStatus.All;
}
